### PR TITLE
 feat: Responsive css, drop shadow fix, settle button disabled fix

### PR DESCRIPTION
**Pull request by Tulun**
Responsive Css fixes
https://app.clubhouse.io/uma-project/story/1926/make-sure-1-1-release-works-on-various-devices-ipad-mobile-various-computer-screens-etc

Update box shadow
https://app.clubhouse.io/uma-project/story/1936/update-shadow-to-match-figma-design-on-modules

Boolean added for this state as it's complicated:
https://app.clubhouse.io/uma-project/story/1893/settle-button-should-be-disabled-in-this-state-in-the-image

### CODE REVIEW

File: components/lsp/LSP.tsx [hunk 1]
```diff
@@ -80,6 +82,15 @@ const LSP: FC<Props> = ({
     }
   }, [data.expirationTimestamp, currentTime, contractState]);
 
+  // Coverage in case there is a transition between states and the price settles.
+  useEffect(() => {
+    if (
+      data.contractState > ContractState.ExpiredPriceRequested &&
+      settleButtonDisabled
+    )
+      setSettleButtonDisabled(true);
```
Comment (daywiss) [inline components/lsp/LSP.tsx:91]: Can you double check this logic, its only being set true if its already true

File: pages/[address].tsx [hunk 2]
```diff
@@ -228,7 +228,7 @@ const SynthPage: React.FC<Props> = ({ data, relatedSynths, change24h }) => {
             <Heading>
               {data.type === "emp"
                 ? data.tokenName
-                : formatLSPName(data.longTokenName)}
+                : formatLSPName(data.longTokenName || "")}
```
Comment (daywiss) [inline pages/[address].tsx:231]: Should this be pairname?

File: components/lsp/LSP.tsx (hunk 1 above)
Comment (Tulun) [reply components/lsp/LSP.tsx:91]: Yup, got it.

File: pages/[address].tsx (hunk 2 above)
Comment (Tulun) [reply pages/[address].tsx:231]: Let's say yes. I'm not 100% sure.

File: pages/[address].tsx [hunk 3]
```diff
@@ -214,6 +214,7 @@ const SynthPage: React.FC<Props> = ({ data, relatedSynths, change24h }) => {
     refetchShortTokenBalance,
   ]);
 
+  console.log("data", data);
```
Comment (daywiss) [inline pages/[address].tsx:217]: Stray log

**Review by daywiss — APPROVED** [review]

**Pull request merged**